cfgs/profile/RVA22S64.yaml

```diff
@@ -0,0 +1,111 @@
+# SPDX-License-Identifier: CC0-1.0
+
+# AUTO-GENERATED FILE. DO NOT EDIT
+# To regenerate, run `./do gen:cfg` in the UDB root directory
+# The data comes from the UDB profile definitions in spec/std/isa/profile/
+
+---
+"$schema": config_schema.json#
+kind: architecture configuration
+type: partially configured
+name: RVA22S64
+description:
+params:
+  MXLEN: 64
+mandatory_extensions:
+- name: A
+  version: "= 2.1"
+- name: C
+  version: "= 2.0"
+- name: D
+  version: "= 2.2"
+- name: F
+  version: "= 2.2"
+- name: I
+  version: "~> 2.1"
+- name: M
+  version: "= 2.0"
+- name: S
+  version: "= 1.12"
+- name: Ssccptr
+  version: "= 1.0"
+- name: Sscounterenw
+  version: "= 1.0"
+- name: Sstvala
+  version: "= 1.0"
+- name: Sstvecd
+  version: "= 1.0"
+- name: Sv39
+  version: "= 1.11"
+- name: Svade
+  version: "~> 1.0"
+- name: Svbare
+  version: "= 1.0"
+- name: Svinval
+  version: "~> 1.0"
+- name: Svpbmt
+  version: "~> 1.0"
+- name: U
+  version: "~> 2.0"
+- name: Za128rs
+  version: "= 1.0"
+- name: Zba
+  version: "~> 1.0"
+- name: Zbb
+  version: "~> 1.0"
+- name: Zbs
+  version: "~> 1.0"
+- name: Zfhmin
+  version: "~> 1.0"
+- name: Zic64b
+  version: "= 1.0"
+- name: Zicbom
+  version: "~> 1.0"
+- name: Zicbop
+  version: "~> 1.0"
+- name: Zicboz
+  version: "~> 1.0"
+- name: Ziccamoa
+  version: "= 1.0"
+- name: Ziccif
+  version: "= 1.0"
+- name: Zicclsm
+  version: "= 1.0"
+- name: Ziccrse
+  version: "= 1.0"
+- name: Zicntr
+  version: "= 2.0"
+- name: Zifencei
+  version: "= 2.0"
+- name: Zihintpause
+  version: "= 2.0"
+- name: Zihpm
+  version: "= 2.0"
+- name: Zkt
+  version: "~> 1.0"
+non_mandatory_extensions:
+- name: Sha
+  version: "~> 1.0"
+- name: Sscofpmf
+  version: "~> 1.0"
+- name: Sstc
+  version: "~> 1.0"
+- name: Ssu64xl
+  version: "= 1.0"
+- name: Sv48
+  version: "~> 1.12"
+- name: Sv57
+  version: "~> 1.12"
+- name: Svnapot
+  version: "~> 1.0"
+- name: V
+  version: "~> 1.0"
+- name: Zfh
+  version: "~> 1.0"
+- name: Zkn
+  version: "~> 1.0"
+- name: Zkr
+  version: "~> 1.0"
+- name: Zks
+  version: "~> 1.0"
```

Comment:
I know this is a generated file, but this seems to exhibit the symptoms of a problem. To my understanding, none of V, Zfh, Zkn, nor Zks should be here.